rdbi-core/src/test/java/com/lithium/dbi/rdbi/RDBITest.java

```diff
@@ -15,40 +15,47 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
 import static org.testng.Assert.assertTrue;
 import static org.testng.Assert.fail;
 
 public class RDBITest {
 
-    interface TestDAO {
+    public interface TestDAO {
```

Comment:
this type of change will probably cause some problems for updating users. if the interface isn't in the `com.lithium.dbi.rdbi` package, we are unable to create a proxy for it unless it is public (previously we could).